implementations/rust/ockam/ockam_command/src/run/config.rs

```diff
@@ -79,26 +77,29 @@ impl Config {
     pub async fn parse_and_run(
         ctx: &Context,
         opts: CommandGlobalOpts,
-        contents: &str,
+        contents: &mut String,
```

Comment:
I think this is better unless you want to return contents back
```
mut contents: String
```

Reply:
Right, good catch! I'll update it